.env.default

```diff
@@ -2,5 +2,6 @@ NODE_ENV=development
 API_KEY=dvl-1510egmf4a23d80342403fb599qd
 PG_DATABASE=opencollective_dvl
 PORT=3060
-WEBSITE_URL=http://localhost:3000
+WEBSITE_URL=http://localhost:13000
```

Comment:
Note that this is wrong when not using Docker, and that this will be overhidden by the Docker environment variables anyway, so unnecessary.

See:
https://github.com/opencollective/opencollective-api/blob/master/docker/docker-compose.yml

Reply:
Ah ok. The reason I noticed this is that emails use this envvar when building URLS, so links in emails were broken when using Docker.

Reply:
@sloria Thank you for the feedback. We will need to check what are the actual priorities between the rules set in the Docker environment and the one set in `.env`.

Reply:
I could be wrong, but I think the file overwrites the values for the application because it's loaded within docker.

I added a little bit more of context on the PR that you referenced @znarf! Thanks a lot! We'll get that running smoothly soon!